docs/source/models/atomic_physics.rst

```diff
@@ -24,15 +24,15 @@ FLYonPIC is based on the [FlyCHK]_ atomic model and currently implements the fol
 
 - electronic collisional de-/excitation
 - spontaneous deexcitation
-- electronic ionization (with ionization potential depression)
+- electronic ionization (with Stewart-Pyatt ionization potential depression)
```

Comment:
This seems like a typo (same as two lines above) 
Please provide a reference here to defend your naming scheme